kedro/ipython/__init__.py

```diff
@@ -2,19 +2,23 @@
 This script creates an IPython extension to load Kedro-related variables in
 local scope.
 """
+
 from __future__ import annotations
 
 import inspect
 import logging
+import os
 import sys
 import typing
 import warnings
 from pathlib import Path
 from typing import Any, Callable
 
-import IPython
+from IPython.core.getipython import get_ipython
```

Comment:
I change the import here, but honestly it doesn't matter if we suppress the mypy warnings. It's meaningless because it is complaining IPython module doesn't export `get_ipython`. If you google around that's basically how everyone use it even on Databricks.

"Technically", this is the module that you can import from. I don't think this is beneficial other than keeping mypy quiet, I made the change anyway.